assets/js/controllers/transactions/transactionsEthereum.controller.js

```diff
@@ -96,11 +96,9 @@ function ethereumTransactionsCtrl ($scope, $uibModal, $state, Wallet, Ethereum,
       case $scope.filterTypes[1]:
         if (Ethereum.legacyAccount) return tx.getTxType([$scope.account, Ethereum.legacyAccount]) === 'sent';
         else return tx.getTxType([$scope.account]) === 'sent';
-        break;
       case $scope.filterTypes[2]:
         if (Ethereum.legacyAccount) return tx.getTxType([$scope.account, Ethereum.legacyAccount]) === 'received';
         else return tx.getTxType([$scope.account]) === 'received';
-        break;
     }
```

Comment:
Are breaks unnecessary if we `return`?

Reply:
That's correct.  It violates the `no-unreachable` rule.  `break` and `return` are synonymous inside a simple switch statement like this.  However, this may not always be the case if the switch statement is being ran inside some sort of an outer loop function.